Vireo_VS/VireoCommandLine.vcxproj

```diff
@@ -96,7 +96,7 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <PlatformToolset>v140</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
-    <CharacterSet>Unicode</CharacterSet>
+    <CharacterSet>MultiByte</CharacterSet>
```

Comment:
I think this would be my only concern, can you elaborate why we need to use Multibyte instead of Unicode.